CHANGELOG.md

```diff
@@ -82,6 +82,8 @@ and this project adheres to
   - [#1538](https://github.com/iovisor/bpftrace/pull/1538)
 - Unify semantics of wildcards in probe listing and attachement
   - [#1549](https://github.com/iovisor/bpftrace/pull/1549)
+- Improve codegen for structs and arrays
+  - [#1705](https://github.com/iovisor/bpftrace/pull/1705)
```

Comment:
printing of structs is a nice feature that deserves a place in the changelog, maybe add two entries for the same pr?

Reply:
yes, definitely, I added a second entry